_includes/install/file-system-perms-twouser_22.md

```diff
@@ -105,12 +105,26 @@ To set ownership and permissions before you install the Magento software:
 
 1.	Log in to your Magento server as, or switch to, the Magento file system owner.
 2.	Enter the following commands in the order shown:
-
-		cd <your Magento install dir>
-		find var generated vendor pub/static pub/media app/etc -type f -exec chmod g+w {} +
-		find var generated vendor pub/static pub/media app/etc -type d -exec chmod g+ws {} +
-		sudo chown -R :<web server group> .
-		chmod u+x bin/magento
+    
+    ```bash
+	cd <magento_root>
+	```
+	
+	```bash
```

Comment:
I have doubts about wrapping every command into a separate tag. However, they should be executed step by step. So, it's ok.

Reply:
Thank you for reviewing, @rogyar .

Regarding @dobooth [comment](https://github.com/magento/devdocs/pull/4274#issuecomment-484224801) such format is ok. 

Thank you!